pfcpiface/config.go

```diff
@@ -86,13 +86,18 @@ type SimModeInfo struct {
 
 // CPIfaceInfo : CPIface interface settings.
 type CPIfaceInfo struct {
-	Peers           []string `json:"peers"`
-	UseFQDN         bool     `json:"use_fqdn"`
-	NodeID          string   `json:"hostname"`
-	HTTPPort        string   `json:"http_port"`
-	Dnn             string   `json:"dnn"`
-	EnableUeIPAlloc bool     `json:"enable_ue_ip_alloc"`
-	UEIPPool        string   `json:"ue_ip_pool"`
+	Peers           []string  `json:"peers"`
+	UseFQDN         bool      `json:"use_fqdn"`
+	NodeID          string    `json:"hostname"`
+	HTTPPort        string    `json:"http_port"`
+	DnnList         []DNNInfo `json:"dnn_list"`
+	EnableUeIPAlloc bool      `json:"enable_ue_ip_alloc"`
+	UEIPPool        string    `json:"ue_ip_pool"`
```

Comment:
Given that you are moving the `UEIPPool` to inside `DNNInfo`, should not it be removed from here?

```suggestion
```